contracts/ServiceNodeRewards.sol

```diff
@@ -52,7 +51,6 @@ contract ServiceNodeRewards is Initializable, Ownable2StepUpgradeable, PausableU
     function initialize(address token_, address foundationPool_, uint256 stakingRequirement_, uint256 maxContributors_, uint256 liquidatorRewardRatio_, uint256 poolShareOfLiquidationRatio_, uint256 recipientRatio_) initializer()  public {
         if (recipientRatio_ < 1) revert RecipientRewardsTooLow();
         IsActive                     = false;
-        nextServiceNodeID            = 1;
```

Comment:
What was the thought behind this one? Wont the add service node functions now try to write to our list sentinel?

Reply:
Just below here a couple of lines down we do `nextServiceNodeID = LIST_SENTINEL + 1;` so it was a double-initialise of the variable that slither detected.

LIST_SENTINEL is 0 initially, so, this'd set it to 1, same effect as the line here